hybrid/decomposers.py

```diff
@@ -316,6 +318,59 @@ def next(self, state, **runopts):
         return state.updated(subproblem=subbqm)
 
 
+class RoofDualityDecomposer(Runnable, traits.ProblemDecomposer):
+    """Selects a subproblem with variables that cannot be fixed by roof duality.
+
+    Roof duality finds a lower bound for the minimum of a quadratic polynomial.
+    It can also find minimizing assignments for some of the polynomial's
+    variables; these fixed variables take the same values in all optimal
+    solutions [BHT]_ [BH]_. A quadratic pseudo-Boolean function can be
+    represented as a network to find the lower bound through network-flow
+    computations. `fix_variables` uses maximum flow in the implication network
```

Comment:
Ref to `fix_variables`, but not clear from where.

Reply:
Oops, that's what I get for copy and pasting...